### PR TITLE
Switch off deeply read test

### PR DESCRIPTION
**Pull request by DanielCliftonGuardian**
## What is the value of this and can you measure success?
Switches off deeply read test
Part of https://github.com/guardian/dotcom-rendering/issues/11672

## Screenshots

N/A

### CODE REVIEW

**Comment by domlander** [comment]
I usually bump commercial at the same time as when turning off a switch in frontend. That way this equivalent file in commercial will have it's audience set to zero at the same time

File: package.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
 		"@emotion/react": "11.11.1",
 		"@emotion/styled": "^10.0.27",
 		"@guardian/ab-core": "7.0.1",
-		"@guardian/commercial": "19.6.0",
+		"@guardian/automat-modules": "^0.3.8",
+		"@guardian/commercial": "19.7.0",
 		"@guardian/core-web-vitals": "6.0.0",
```
Comment (abeddow91) [inline package.json:33]: Are these changes necessary as part of this PR?
